_layouts/base.html

```diff
@@ -48,18 +57,18 @@
             <a href="https://devedmonton.com">DES</a> with &hearts;.
           </li>
           <li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
-          <li><a href="./policies/privacy-policy.html">Privacy Policy</a></li>
+          <li><a href="/policies/privacy-policy.html">Privacy Policy</a></li>
```

Comment:
I had to time-box how deep I was getting into Jekyll's docs, but pretty sure this is not ideal.

Reply:
There are ways to get things like the base url using liquid template functions, but honestly I don't think this is a big issue. We're only viewing the rendered content anyway.

I'd say leave this as is and we can worry about it later if it becomes an issue. I'd say this is accepted practice on all the other sites I've worked on with a render pipeline for content so not something we need to complicate here.

Reply:
See above comment about `relative_url`